db/notification/notification-dismiss.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
Currently we have a notifications-service folder. Could you move this file into notifications-service folder?